openslides/utils/rest_api.py

```diff
@@ -160,6 +162,14 @@ def get_serializer_class(self) -> Type[Serializer]:
             serializer_class = super().get_serializer_class()  # type: ignore
         return serializer_class
 
+    def finalize_response(self, request, response, *args, **kwargs):
+        """
+        Just remove all response data so nobody may get unrestricted data.
+        """
+        response = super().finalize_response(request, response, *args, **kwargs)
```

Comment:
I think this method is also called on get requests.

If we want to keep the rest api, they have to see the data.

Reply:
Yes, If we do this, we can disable the rest api. I'm not a fan of just removing the data, but this may be the simplest solution.

We may provide a list of keys (default an empty list) with keys to remove on responses. So users can have `['session_auth_hash']`. Here, the all entries with keys in this blacklist will be removed. This should be a simple solution, too, because you just have to specify keys for each model to be blacklisted.

Reply:
But how can I access the REST API with the new client? /rest does not work.

Reply:
You need to use the old port (8000).

Reply:
Ok, done.

Reply:
Are there other http methods that needs to return data? What about OPTIONS requests?

Is it necessary now to go throw all writing requests and look to the client if it needs a response?

Reply:
I think we do not want to support OPTIONS requests any more.

And yes: Client delevopers should define which information they want to have as a result of writing operations. So I strongly recommend not to use a blacklist but a whitelist.

Reply:
Could you add one example to this pull request, how a rest-view can explicit return data?